server/deskroom/app.py

```diff
@@ -22,6 +23,8 @@ class State(TypedDict):
 @asynccontextmanager
 async def lifespan(app: FastAPI) -> AsyncIterator[State]:
     async with worker_lifespan() as arq_pool:
+        logger.info("Downloading Prompts")
+        download_prompts()
```

Comment:
```suggestion
        if not settings.is_local() and not settings.is_test():
            download_prompts()
```
로컬과 테스트 환경에서는 스킵하는 게 좋지 않을까요